data_inconsistencies.py

```diff
@@ -32,17 +33,37 @@ def get_inconsistencies():
     return incons
 
 
+def print_statistics(file_name, incons, n_incons):
+    f = utils.get_all_images()
+    total_img = len(f)
+    firefox = []
```

Comment:
Define n_incons here since you have incons here so the signature of the function will be cleaner